packages/odyssey/src/scss/vendors-ext/_choices-ext.scss

```diff
@@ -0,0 +1,453 @@
+    color: $white;
+    font-size: ms(-1);
+    font-weight: 500;
+    word-break: break-all;
+    vertical-align: middle;
+
+    &[data-deletable] {
+      padding-right: 5px;
+    }
+
+    [dir='rtl'] & {
+      margin-right: 0;
+      margin-left: 3.75px;
+    }
+
+    &.is-ods-select-highlighted {
+      border: 1px solid darken(cv('action', 'base'), 10%);
+      background-color: darken(cv('action', 'base'), 5%);
+    }
+
+    .is-ods-select-disabled & {
+      border: 1px solid darken(cv('gray', '000'), 35%);
+      background-color: darken(cv('gray', '000'), 25%);
+    }
+  }
+}
+
+.ods-select--list--dropdown {
+  visibility: hidden;
+  position: absolute;
+  z-index: 1;
+  top: 100%;
+  width: 100%;
+  overflow: hidden;
+  border: 1px solid $base-border-color;
+  border-top-width: 0;
+  border-bottom-right-radius: $base-border-radius;
+  border-bottom-left-radius: $base-border-radius;
+  background-color: $white;
+  word-break: break-all;
+  will-change: visibility;
+
+  &.is-ods-select-active {
+    visibility: visible;
+  }
+
+  .is-ods-select-focused & {
+    box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base'));
+    clip-path: inset(0 -4px -4px -4px);
+  }
+
+  .is-ods-select-open & {
+    border-color: cv('action', 'base');
+  }
+
+  .is-ods-select-flipped & {
+    top: auto;
+    bottom: 100%;
+    border-bottom: 0;
+    border-top-width: 1px;
+    border-radius: $base-border-radius $base-border-radius 0 0;
+  }
+
+  .is-ods-select-flipped.is-ods-select-focused & {
+    clip-path: inset(-4px -4px 0 -4px);
+  }
+
+  .ods-select--list {
+    position: relative;
+    max-height: 300px;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    will-change: scroll-position;
+  }
+
+  .ods-select--item {
+    position: relative;
+    padding: $small-spacing;
+    font-size: ms(0);
+
+    [dir='rtl'] & {
+      text-align: right;
+    }
+
+    &:empty {
+      display: none;
+    }
+  }
+
+  .ods-select--item--selectable {
+    @media (min-width: 640px) {
+      padding-right: 100px;
+
+      &::after {
+        content: attr(data-select-text);
+        position: absolute;
+        top: 50%;
+        right: $small-spacing;
+        transform: translateY(-50%);
+        opacity: 0;
+        font-size: ms(-1);
+      }
+
+      [dir='rtl'] & {
+        padding-right: $small-spacing;
+        padding-left: 100px;
+        text-align: right;
+
+        &::after {
+          right: auto;
+          left: $small-spacing;
+        }
+      }
+    }
+
+    &.is-ods-select-highlighted {
+      background-color: cv('action', 'lightest');
+
+      &::after {
+        opacity: 0.5;
+      }
+    }
+  }
+
+  [data-invalid] .is-ods-select-focused & {
+    box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base'));
+  }
+
+  [data-invalid] .is-ods-select-open & {
+    border-color: cv('danger', 'base');
+  }
+}
+
+.ods-select--item {
+  cursor: default;
+}
+
+.ods-select--item--selectable {
+  //cursor: pointer;
```

Comment:
nit: remove?

Reply:
Same as others above - I want to preserve the selector since it comes from Choices.js, but there aren't any applicable properties right now. (I could add a rule exception here to allow empty selectors in stylelint, but this feels like a better reminder that this property itself needs to be checked on later.)